fcp-20190401-ci_policy.md

```diff
@@ -6,85 +6,85 @@ state: draft
+system is configured to send failure or instability notifications to the committer
+and the dev-ci@freebsd.org mailing list.  Ideally the original committer responds to
+these notifications by committing a fix or revert themselves.  If these
 reports are not addressed in a timely manner a number of negative effects
 compound:
 
  - Other issues may be hidden by this failure
  - Development on affected platforms may be blocked
  - Downstream users may struggle to find a usable version
 
-While developers are usually very responsive to reports of build failures they
-may not always know about this issue, or may not have the time to immediately
+While developers are usually very responsive to reports of build failures, they
+may not always know how to resolve a specific issue promptly, or may not have the time to immediately
 address it.
 
 Other developers do not always feel empowered to revert such commits.
 
 ## Proposed Solution
 
 We propose a policy explicitly empowering developers to revert commits which
-cause compilation problems, test failures or panics during tests.
+cause compilation problems, test failures, or panics during tests.
 
-We describe the recommended steps to take, and notifications to send before
-doing so, as well as the recommended timeout periods.
+We describe the recommended steps to take, notifications to send,
+and recommended timeout periods.
 
 ### Notification
 
-When a failure occurs in the FreeBSD's CI system (ci.freebsd.org)'s
-non-experimental jobs [1], anyone (committers and non-committers alike) is
-encouraged to investigate this. The commit which triggered this failure should
-be identified, and the author contacted.
+Everyone (committers and non-committers alike) are
+encouraged to investigate failures in non-experimental jobs [1].
+The investigator should prepare a report identifying the commit which triggered
+the failure as well as details about the failure (for example, compile errors or
+output from failing tests).  The investigator should
+The commit which triggered this failure should send this report to all of the following:
 
-The notification should be sent to all of the following:
- - The original author
+ - The author of the triggering commit
  - Other people, if any, who've touched the relevant code after the triggering
    commit
- - The dev-ci@freebsd.org list, so that duplicate notifications can be avoided
-   as much as possible
+ - The dev-ci@freebsd.org list (to avoid duplicate notifications)
 
 ### Waiting period
 
-Once the notification has been sent the author of the problematic commit should
+Once the notification has been sent, the author of the problematic commit should
 be given a reasonable amount of time to respond.
 This time will vary depending on the severity of the problem and how many
 platforms it affects:
 
- - If a commit broke compilation on all platforms it can be reverted
+ - If a commit broke compilation on all platforms, it can be reverted
    immediately
```

Comment:
Immediately is kind of vague.  I could send a reply and then revert without waiting for a reply.  I think it would be more useful to pick an actual time frame like "15 minutes".